polling_stations/apps/file_uploads/tests/test_api.py

```diff
@@ -14,6 +14,7 @@
 
 
 @patch("file_uploads.models.Upload.make_pull_request", successful_upload_pull_request)
+@override_settings(API_AUTH_TOKENS=["superuser-key"])
```

Comment:
Maybe redundant given changes in `polling_stations/settings/testing.py`?

Reply:
True! This was more to prove that we didn't need to alter the existing tests for the new system to work...I thought there was value in mostly unchanged test code as validation